antlr4ide.core/src/main/java/com/github/jknack/antlr4ide/console/Console.java

```diff
@@ -0,0 +1,48 @@
+package com.github.jknack.antlr4ide.console;
+
+/**
+ * Console and logger for showing feedback to users.
+ */
+public interface Console {
```

Comment:
Maybe some day we want to change to another logging system? To prevent problems with such situations, I would advertise for using this interface as a wrapper around the logging-system. But of course, you are right that using the log4j-interfaces and classes provide more functionality than this simplified one.